src/Controller/Customer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Customers as CustomerEntity;
+use App\Entity\Users as UserEntity;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+
+class Customer extends AbstractController
+{
+
+    public function customers(Request $request)
+    {
+        $customers = $this->getDoctrine()->getRepository('App:Customers')->findAll();
```

Comment:
What you think about using `#getRepository(CustomerEntity::class)` instead of `#getRepository('App:Customers')`? This way we can use our IDE's refactor feature without problems.

Reply:
Yes, lack of attention. I'm make the correction !